setup.py

```diff
@@ -89,6 +110,14 @@ def findRequirements():
   requirementsPath = os.path.join(REPO_DIR, "requirements.txt")
   requirements = parse_file(requirementsPath)
 
+  if nupicBindingsPrereleaseInstalled():
+    # User has a pre-release version of nupic.bindings installed, which is only
```

Comment:
If I am not mistaken, the solution assumes that we're not going to deploy pre-release versions of nupic.bindings to PyPi. Is this a realistic assumption? My understanding is that we're planning to make more frequent deployments to PyPi. Would they not include pre-release versions?

cc @scottpurdy